bakerydemo/base/jinja2tags.py

```diff
@@ -0,0 +1,61 @@
+from django.template.context_processors import csrf
+from django.template.defaultfilters import (
+    cut,
+    date,
+    linebreaks,
+    pluralize,
+    slugify,
+    truncatewords,
+    urlencode,
+)
+from django.contrib.staticfiles.storage import staticfiles_storage
+from django.urls import reverse
+from jinja2 import pass_context
+from jinja2.ext import Extension
+
+from wagtail.contrib.search_promotions.templatetags.wagtailsearchpromotions_tags import (
+    get_search_promotions,
+)
```

Comment:
This is the only bakerydemo feature for which we don’t have Jinja support directly in Wagtail.